src/Workspaces/Core/Desktop/Workspace/SQLite/SQLite.cs

```diff
@@ -0,0 +1,4003 @@
+﻿//
```

Comment:
Not part of the review.  this is only so we can experiment with this library until we get an appropriate signing story worked out.